.github/workflows/ci-workflow.yml

```diff
@@ -10,56 +10,6 @@ on:
   pull_request:
 
```

Comment:
ci-workflow.yml should be renamed (not explicit)

Reply:
Done